l10n_br_zip/models/l10n_br_zip.py

```diff
@@ -20,6 +20,7 @@ class L10nBrZip(models.Model):
 
     zip_code = fields.Char('CEP', size=8, required=True)
     street_type = fields.Char('Tipo', size=26)
+    number_range = fields.Char('Range', size=200)
```

Comment:
You should rename this field to zip_complement, because some cases the information on complemento2 has street number range, but in others cases has other kind of information like:

(enderecoERP){
   bairro = "Jardim das Bandeiras"
   cep = "05436170"
   cidade = "São Paulo"
   complemento2 = "- lado ímpar"
   end = "Rua Abegoaria"
   uf = "SP"
 }

(enderecoERP){
   bairro = "Jardim Moreno"
   cep = "08430853"
   cidade = "São Paulo"
   complemento2 = "(Cj Hab Baltazar Cisneiro)"
   end = "Rua Aba"
   uf = "SP"
 }